pygradle-plugin/src/integTest/groovy/com/linkedin/gradle/python/plugin/PyGradleTestBuilder.groovy

```diff
@@ -24,7 +24,7 @@ class PyGradleTestBuilder {
             | max-line-length = 160
             |
             | [pytest]
-            | addopts = --ignore build/ --ignore dist/
+            | addopts = --ignore build/ --ignore dist/ --junitxml TEST-pytest.xml
```

Comment:
I don't think this will help CI systems in any way. This is the test for {py}gradle itself.
Let's talk offline about your needs and how to best solve them.
Could you please delete this branch from the project?
We prefer that the development for pull requests be done on the forks of the project, rather than directly on the master repo.